src/Cech_complex/include/gudhi/Cech_complex/Cech_kernel.h

```diff
@@ -0,0 +1,107 @@
+/*    This file is part of the Gudhi Library - https://gudhi.inria.fr/ - which is released under MIT.
+ *    See file LICENSE or go to https://gudhi.inria.fr/licensing/ for full license details.
+ *    Author(s):       Hind Montassif
+ *
+ *    Copyright (C) 2021 Inria
+ *
+ *    Modification(s):
+ *      - YYYY/MM Author: Description of the modification
+ */
+
+#ifndef CECH_KERNEL_H_
+#define CECH_KERNEL_H_
+
+#include <CGAL/Epeck_d.h> // for #include <CGAL/NewKernel_d/KernelD_converter.h>
+
+#include <cmath>  // for std::sqrt
+#include <vector>
+
+namespace Gudhi {
+
+// namespace cech_complex {
+
+/** @brief Compute the radius of the minimal enclosing ball between Points given by a range of coordinates.
+ * The points are assumed to have the same dimension. */
+template<typename Kernel>
+class Minimal_enclosing_ball_radius {
```

Comment:
This name and the comments are misleading, from what I can see in the code it computes the circumradius (points must be on the sphere), not an enclosing ball (points are allowed to be inside the ball). Or if the name and comments are right, then the code is wrong ;-)
It seems that this class is meant to be private and only used for testing / debugging?

Reply:
> It seems that this class is meant to be private and only used for testing / debugging?

Yes indeed, do you have something specific in mind?